crowdfunding/projects/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+
+from .models import Project, Pledge
+
+from users.serializers import CustomUserSerializer
+
+class PledgeSerializer(serializers.ModelSerializer):
+    supporter = serializers.SerializerMethodField()
+    class Meta:
+        model = Pledge
+        fields = '__all__'
+        read_only_fields = ['id', 'supporter']
+
+    def get_supporter(self, instance):
+        if instance.anonymous:
+            return "anonymous"
```

Comment:
i would have returned None rather than anonymous,
returning None is a more honest response from a data perspective, as you may have a user named anonymous